.travis.yml

```diff
@@ -9,6 +9,9 @@ sudo: false
 os: osx
 osx_image: xcode11.3
 
+before_script:
+  - export JAVA_HOME=/Library/Java/JavaVirtualMachines/openjdk-13.0.1.jdk/Contents/Home
```

Comment:
`jdk: openjdk11` can be removed to avoid confusion.

Reply:
done